mozilla-release/ipc/app/module.ver

```diff
@@ -4,3 +4,5 @@ WIN32_MODULE_PRODUCTVERSION_STRING=@MOZ_APP_VERSION@
 WIN32_MODULE_DESCRIPTION=Plugin Container for @MOZ_APP_DISPLAYNAME@
 WIN32_MODULE_PRODUCTNAME=@MOZ_APP_DISPLAYNAME@
 WIN32_MODULE_NAME=@MOZ_APP_DISPLAYNAME@
+WIN32_MODULE_FILEVERSION=@MOZ_APP_WINVERSION@

```

Comment:
IPC usually stands for "inter process communication". In this case, seems, like version info goes to a helper application, hosting plugins.